DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,7 @@ public function getConfigTreeBuilder()
 
         $rootNode
             ->children()
+                ->scalarNode('default_formatter')->end() // TODO: make it required when the major version is changed
```

Comment:
Everything is done the same as in MediaBundle except from a few BC preserving code.